eng/testing/tests.mobile.targets

```diff
@@ -106,11 +106,30 @@
     <!-- This app is now can be consumed by xharness CLI to deploy on a device or simulator -->
   </Target>
 
+  <UsingTask TaskName="WasmAppBuilder"
+             AssemblyFile="$(ArtifactsBinDir)\WasmAppBuilder\$(MonoConfiguration)\$(NetCoreAppCurrent)\publish\WasmAppBuilder.dll" />
```

Comment:
refactor the path into a variable, like AppleAppBuilderTasksAssemblyPath:
https://github.com/dotnet/runtime/blob/b6ef0a54ad0dbe016ec63850e3a70bfacb9bde09/Directory.Build.props#L55

Reply:
All those variables include 'Debug' as the configuration, is that right ?

Reply:
Yes, they are built here which hardcodes Debug:
https://github.com/dotnet/runtime/blob/32fdcaf5eeb890c00f2a5310cc36b24cc1025884/tools-local/tasks/tasks.proj#L32-L34